src/main/java/chess/controller/ConsoleChessController.java

```diff
@@ -0,0 +1,51 @@
+package chess.controller;
+
+import chess.chessgame.ChessGame;
+import chess.chessgame.MovingPosition;
+import chess.view.OutputView;
+
+import java.util.StringTokenizer;
+
+import static chess.view.InputView.inputCommand;
+import static chess.view.OutputView.printStartMessage;
+
+public class ConsoleChessController {
+
+    public void run() {
+        ChessGame chessGame = new ChessGame();
+        printStartMessage();
+        while (!chessGame.isFinished()) {
+            playTurn(chessGame);
+        }
+        OutputView.printScore(chessGame.computeScore());
+    }
+
+    private void playTurn(ChessGame chessGame) {
+        String input = inputCommand();
+        StringTokenizer commands = new StringTokenizer(input);
+        String command = commands.nextToken();
+
+        if (command.equals("start")) {
```

Comment:
```suggestion
        if ("start"equals(command.)) {
```
모든 문자열 equals를 위와 같이 바꿔보세요.
그리고 왜 바꾸라고 했을지 고민해보아요

Reply:
command.equals("start")의 경우 command가 null이라면 NullPointException이 발생할 위험이 있습니다. 하지만 이를 "start".equals(command)로 수정한다면  command가 null이라도 false가 return될 뿐, 예외가 발생할 위험은 없습니다. 수정 완료 했습니다! 해당 부분은 전혀 알지 못했던 부분입니다. 감사합니다 :)